Content.Client/Entry/EntryPoint.cs

```diff
@@ -191,6 +192,7 @@ public override void PostInit()
             overlayMgr.AddOverlay(new CircleMaskOverlay());
             overlayMgr.AddOverlay(new FlashOverlay());
             overlayMgr.AddOverlay(new RadiationPulseOverlay());
+            overlayMgr.AddOverlay(new BlindOverlay());
```

Comment:
This should be added or removed dynamically.

Reply:
How would you do that and why do none of the other overlays do that?